### PR TITLE
Remove MYRIAD/VAD-M related code as their plugin support is deprecated by OpenVINO

### PR DESCRIPTION
**Pull request by sspintel**
Verified that changes don't break build.

### CODE REVIEW

File: dockerfiles/Dockerfile.openvino-rhel8 [hunk 1]
```diff
@@ -27,20 +27,6 @@ RUN yum install -y yum-utils autoconf automake libtool unzip udev wget zlib-deve
     wget https://www.python.org/ftp/python/3.8.9/Python-3.8.9.tgz && tar xvf Python-3.8.9.tgz && rm -rf Python-3.8.9.tgz && \
     cd Python-3.8*/ && ./configure && make && make install && \
     cd ../ &&  mkdir -p /usr/bin/Python38 && ln -s Python-3.8.9/ /usr/bin/Python38 && ln -s /usr/bin/pip3 /usr/bin/pip && \
-# libusb1.0.22
```
Comment (sfatimar) [inline dockerfiles/Dockerfile.openvino-rhel8:30]: Please dont make changes in dockerfiles. We no longer maintain them and any change in them trigger a long dicussion with msft

File: tools/ci_build/github/linux/docker/Dockerfile_manylinux2014_openvino_multipython [hunk 2]
```diff
@@ -18,21 +18,6 @@ ENV PYTHON_EXE="/opt/python/$PYTHON_VER_PATH/bin/python$PYTHON_VERSION"
 
 RUN yum -y install wget git
 
-# libusb1.0.22
```
Comment (sfatimar) [inline tools/ci_build/github/linux/docker/Dockerfile_manylinux2014_openvino_multipython:21]: Same comment here, no changes

**Review by sfatimar — COMMENTED** [review]
Please look at comments

**Comment by MaajidKhan** [comment]
LGTM. Approved. Merged.

**current status:**
[openvino-ep-ort-5.0](https://github.com/intel/onnxruntime/tree/openvino-ep-ort-5.0) branch is in sync with MSFT master and has the changes from the branch - [sp/remove-myd-hddl-code](https://github.com/intel/onnxruntime/tree/sp/remove-myd-hddl-code) merged.

**Pull request merged**